apis/ml-projects/ml-projectsapi.yaml

```diff
@@ -0,0 +1,1863 @@
+            application/json:
+              schema:
+                $ref: '#/components/schemas/500Error'
+      deprecated: false
+      security:
+        - bearer: []
+  /userProjects/mlprojects/v1/add:
+    post:
+      tags:
+      - userProjects APIs
+      summary: Add project.
+      description: |-
+        This API is associated with creating project on the Sunbird Platform.
+          - The endpoint for **UserProjects add** is `/userProjects/mlprojects/v1/add`
+          - It is mandatory to provide values for parameters marked with *. 
+          - Mandatory fields cannot be null or empty.
+      operationId: add
+      parameters:
+        - name: X-Authenticated-User-Token
+          in: header
+          description: Represents the authorized User ID to execute the API.
+          required: true
+          schema:
+            type: string
+        - name: Authorization
+          in: header
+          description: >-
+            To make use of the API, you require authorization. Raise a request
+            to the administrator for the use of the API. You will receive the
+            authorization key. Specify the key received, here.
+          required: true
+          schema:
+            type: string
+      requestBody:
+        required: true
+        description: >-
+          'The body parameter refers to the format of the request, the request
+          body (payload) takes the following media types to make the payload:'
+           
+           - JSON 
+           
+          The body is the representation of the resource object for creating
+          project (an array of parameters which describes the project)
+
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/userprojectsapiAddRequestBody"
+      responses:
+        200:
+          description: "Add project Successful Response"
+          content:
+            application/json:
+              schema:
+                  $ref: '#/components/schemas/userprojectsapiAddResponse'
+        400:
+          description: 'BAD REQUEST. The operation failed ! The possible reason for failure is that you may have missed providing input for a mandatory parameter.'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/400ErrorTitleMissing'
+        500:
+          description: "'INTERNAL SERVER ERROR. We track these errors automatically and try to set it right at the earliest. Try refreshing the page. If the problem persists contact us at info@sunbird.org.'"
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/500Error'
+      deprecated: false
+      security:
+        - bearer: []
+  /userProjects/mlprojects/v1/sync/{projectId}:
+    post:
+      tags:
+      - userProjects APIs
+      summary: Sync project.
+      description: |-
+        This API is used to sync the project details.
```

Comment:
@priyanka-TL  what does sync project sync means? what are the entity /databases across project is synced?